bundle/edu.gemini.itc.shared/src/main/scala/edu/gemini/itc/shared/ItcService.scala

```diff
@@ -53,4 +59,10 @@ object ItcService {
 
   type Result = Validation[List[String], ItcResult]
 
+  /** Performs an ITC call on the given host. */
+  def calculate(kc: KeyChain, peer: Peer, source: SourceDefinition, obs: ObservationDetails, cond: ObservingConditions, tele: TelescopeDetails, ins: InstrumentDetails): Future[Result] =
+    TrpcClient(peer).withKeyChain(kc) future { r =>
+      r[ItcService].calculate(source, obs, cond, tele, ins)
+    }
```

Comment:
Perhaps we don't need the keychain?  Do we do some kind of permission checking on the server side?  It seems like this is all public information that you can get via the web service w/o an account or anything anyway?